<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeshuzhiAdd</class>
 <widget class="QDialog" name="TeshuzhiAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="7,1">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_17">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 215, 207);</string>
     </property>
     <property name="title">
      <string>特殊值</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_50" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>18</pointsize>
            <weight>7</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(173, 127, 168);
color: rgb(0, 0, 0);
font: 57 18pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>因子编码</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lineEdit_Code">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	
	background-color: rgb(211, 215, 207);
	opacity: 1;
	border-radius: 0px;
	combobox-popup: 0;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
	font: 57 15pt &quot;Ubuntu&quot;;
	font-weight:140;
	
	
}

/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
    height: 50px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	font: 57 15pt &quot;Ubuntu&quot;;
	font-weight:140;
}

/* 下拉框样式 */
QComboBox::drop-down {
    subcontrol-origin: padding;   /* 子控件在父元素中的原点矩形。如果未指定此属性，则默认为padding。 */
    subcontrol-position: top right;   /* 下拉框的位置（右上） */
    width: 50px;   /* 下拉框的宽度 */

    border-left-width: 1px;   /* 下拉框的左边界线宽度 */
    border-left-color: darkgray;   /* 下拉框的左边界线颜色 */
    border-left-style: solid;   /* 下拉框的左边界线为实线 */
    border-top-right-radius: 3px;   /* 下拉框的右上边界线的圆角半径（应和整个QComboBox右上边界线的圆角半径一致） */
    border-bottom-right-radius: 3px;   /* 同上 */

	image: url(:/images/dropdown.png);

	padding: 2px 2px 2px 2px;
}

QScrollBar:vertical {
      border: 2px solid grey;
      width: 50px;
	   background-color: #d0d2d4;   /* 空白区域的背景色*/
}//加宽滑动条按钮

QComboBox::drop-down:hover 
{　　　
background: yellow;　
}
/* 下拉箭头样式 */ 
QComboBox::down-arrow
 {　　
width: 20px; /* 下拉箭头的宽度（建议与下拉框drop-down的宽度一致） */ 　　
background: transparent; /* 下拉箭头的的背景色 */ 　　padding: 0px 0px 0px 0px; /* 上内边距、右内边距、下内边距、左内边距 */　
image: url(:/images/dropdown.png);　
} /* 点击下拉箭头 */ 

QComboBox::down-arrow:on 
{ 　　
image: url(:/images/dropdown.png);
/* 显示上拉箭头 */
 } 



QComboBox QAbstractScrollArea QScrollBar::handle:vertical {
    border-radius: 5px;   /* 圆角 */
    background: rgb(160,160,160);   /* 小方块的背景色深灰lightblue */
}

QComboBox QAbstractScrollArea QScrollBar::handle:vertical:hover {
    background: rgb(90, 91, 93);   /* 越过小方块的背景色yellow */
}

 

 
</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_71" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_76">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>18</pointsize>
            <weight>7</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(173, 127, 168);
color: rgb(0, 0, 0);
font: 57 18pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>因子名称</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_72" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_77">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>18</pointsize>
            <weight>7</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(173, 127, 168);
color: rgb(0, 0, 0);
font: 57 18pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>特殊值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Key">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_73" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_78">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>18</pointsize>
            <weight>7</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(173, 127, 168);
color: rgb(0, 0, 0);
font: 57 18pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>显示内容</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lineEdit_Val">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	
	background-color: rgb(211, 215, 207);
	opacity: 1;
	border-radius: 0px;
	combobox-popup: 0;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
	font: 57 15pt &quot;Ubuntu&quot;;
	font-weight:140;
	
	
}

/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
    height: 50px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	font: 57 15pt &quot;Ubuntu&quot;;
	font-weight:140;
}

/* 下拉框样式 */
QComboBox::drop-down {
    subcontrol-origin: padding;   /* 子控件在父元素中的原点矩形。如果未指定此属性，则默认为padding。 */
    subcontrol-position: top right;   /* 下拉框的位置（右上） */
    width: 50px;   /* 下拉框的宽度 */

    border-left-width: 1px;   /* 下拉框的左边界线宽度 */
    border-left-color: darkgray;   /* 下拉框的左边界线颜色 */
    border-left-style: solid;   /* 下拉框的左边界线为实线 */
    border-top-right-radius: 3px;   /* 下拉框的右上边界线的圆角半径（应和整个QComboBox右上边界线的圆角半径一致） */
    border-bottom-right-radius: 3px;   /* 同上 */

	image: url(:/images/dropdown.png);

	padding: 2px 2px 2px 2px;
}

QScrollBar:vertical {
      border: 2px solid grey;
      width: 50px;
	   background-color: #d0d2d4;   /* 空白区域的背景色*/
}//加宽滑动条按钮

QComboBox::drop-down:hover 
{　　　
background: yellow;　
}
/* 下拉箭头样式 */ 
QComboBox::down-arrow
 {　　
width: 20px; /* 下拉箭头的宽度（建议与下拉框drop-down的宽度一致） */ 　　
background: transparent; /* 下拉箭头的的背景色 */ 　　padding: 0px 0px 0px 0px; /* 上内边距、右内边距、下内边距、左内边距 */　
image: url(:/images/dropdown.png);　
} /* 点击下拉箭头 */ 

QComboBox::down-arrow:on 
{ 　　
image: url(:/images/dropdown.png);
/* 显示上拉箭头 */
 } 



QComboBox QAbstractScrollArea QScrollBar::handle:vertical {
    border-radius: 5px;   /* 圆角 */
    background: rgb(160,160,160);   /* 小方块的背景色深灰lightblue */
}

QComboBox QAbstractScrollArea QScrollBar::handle:vertical:hover {
    background: rgb(90, 91, 93);   /* 越过小方块的背景色yellow */
}

 

 
</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TeshuzhiAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TeshuzhiAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
